example/android/build.gradle

```diff
@@ -2,10 +2,11 @@
 
 buildscript {
     ext {
+        kotlinVersion = "1.3.0"
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 18
         compileSdkVersion = 28
-        targetSdkVersion = 28
```

Comment:
why did this need to be downgraded?

Reply:
This is what fixed the issue for running the detox tests in android-release. When using `targetSdkVersion = 28` the app was getting installed but the tests weren't running.